src/base/static/components/atoms/feedback.js

```diff
@@ -6,6 +6,21 @@ import emitter from "../../utils/emitter";
 
 import "./feedback.scss";
 
+const Swatch = props => {
+  return (
+    <span
+      className="mapseed__swatch"
+      style={{
+        backgroundColor: props.color,
+      }}
+    />
+  );
+};
```

Comment:
Also, we are only using this component in one place - the MapLegendItem. What's the point of making this an atom?

Reply:
I think defining a `ColorSwatch` component within `MapLegendItem` would be best, since that's the only place where we are using the component anyway. It doesn't seem likely that we would use this component anywhere else as well. You can define like so (as described above):

```javascript
const ColorSwatch = styled("span")(props => ({
```